src/configuration-store/configuration-store.ts

```diff
@@ -41,6 +41,6 @@ export interface ISyncStore<T> {
 export interface IAsyncStore<T> {
   isInitialized(): boolean;
   isExpired(): Promise<boolean>;
-  getEntries(): Promise<Record<string, T>>;
```

Comment:
@felipecsl you had a review comment to rename `getAll` to `entries`. Ok for me to also rename `getEntries` to `entries` for consistency?